packages/entities/entities-plugins/vite.config.ts

```diff
@@ -1,4 +1,4 @@
-import sharedViteConfig, { getApiProxies, sanitizePackageName } from '../../../vite.config.shared'
+import sharedViteConfig, { getApiProxies,sanitizePackageName } from '../../../vite.config.shared'
```

Comment:
Shall we keep the space?
```suggestion
import sharedViteConfig, { getApiProxies, sanitizePackageName } from '../../../vite.config.shared'
```